cmd/state-remote-installer/main.go

```diff
@@ -194,6 +194,8 @@ func execute(out output.Outputer, prompt prompt.Prompter, cfg *config.Instance,
 	availableUpdate, err := checker.CheckFor(channel, params.version)
 	if err != nil {
 		return errs.Wrap(err, "Could not retrieve install package information")
+	} else if availableUpdate == nil {
+		return locale.NewError("remote_install_no_available_update", "Could not find package to install. Please try again later.")
```

Comment:
```suggestion
		return locale.NewError("remote_install_no_available_update", "Could not find installer to download. This could be due to networking issues or temporary maintenance. Please try again later.")
```